src/services/AccountsService.ts

```diff
@@ -21,7 +21,6 @@ export abstract class AccountsService {
      * One account object is returned.
      *
      * @returns Account The response returns an account
-     * @throws ApiError
```

Comment:
This is unexpected?

Reply:
Forgot to mention this... 🙏 The previous service generation had this, which was untrue when we moved to `Result` but wasn't being removed. The new client generation has it fixed.